File: code/intermediate/aggregation/Aggregration.js
function Address(streetAddress, city, state, pincode) {
    this.streetAddress = streetAddress;
    this.city = city;
    this.state = state;
    this.pincode = pincode;
}

function Home(houseNo, address) {
    this.houseNo = houseNo;
    this.address = address;
    this.printHomeInfo = function () {
        return `${this.houseNo}, ${this.address.streetAddress}, ${this.address.city} ,${this.address.state}, ${this.address.pincode}`;
    }
}
var home = new Home('414', new Address("Street Address", "City", "State", "547766"));
console.log(home.printHomeInfo());

File: code/intermediate/aggregation/Aggregration.py
class Address:
    def __init__(self, street_address, city, state, pin_code):
        self.street_address = street_address
        self.city = city
        self.state = state
        self.pin_code = pin_code


class Home:
    def __init__(self, house_no, address):
        self.house_no = house_no
        self.address = address

    def printHomeInfo(self):
        return f"{self.house_no}, {self.address.street_address}"


h = Home("414", Address("Street Address", "City", "State", "345566"))
print(h.printHomeInfo())
